src/credentials/CHIPCert.cpp

```diff
@@ -153,20 +153,15 @@ CHIP_ERROR ChipCertificateSet::LoadCert(const uint8_t * chipCert, uint32_t chipC
     uint64_t tag;
 
     reader.Init(chipCert, chipCertLen);
-    reader.ImplicitProfileId = Protocols::OpCredentials::Id.ToTLVProfileId();
 
     err = reader.Next();
     SuccessOrExit(err);
 
     type = reader.GetType();
     tag  = reader.GetTag();
 
-    VerifyOrExit(
-        (type == kTLVType_Structure &&
-         (tag == ProfileTag(Protocols::OpCredentials::Id.ToTLVProfileId(), kTag_ChipCertificate) || tag == AnonymousTag)) ||
-            (type == kTLVType_Array &&
-             (tag == ProfileTag(Protocols::OpCredentials::Id.ToTLVProfileId(), kTag_ChipCertificateArray) || tag == AnonymousTag)),
-        err = CHIP_ERROR_UNEXPECTED_TLV_ELEMENT);
+    VerifyOrExit((type == kTLVType_Structure || type == kTLVType_Array) && (tag == AnonymousTag),
```

Comment:
This was explicitly supporting both the old (pre-spec-change) option and the new one because we had both floating around.  What has changed to allow us to drop the old version?

@pan-apple

Reply:
I believe only the test vectors were using the old protocol-specific tag. I have updated all of them to use Anonymous tag.

Reply:
The conversion of x509 certs to CHIP Cert is also updated as part of this PR. That updates the certs to use Anonymous tag instead of older `kTag_ChipCertificate`, `kTag_ChipCertificateArray` tags.